h2o-bindings/bin/gen_R.py

```diff
@@ -40,13 +40,20 @@ def gen_module(schema, algo, module):
             continue
         if algo == "naivebayes":
             if param["name"] == "min_sdev":
-                yield "#' @param threshold The minimum standard deviation to use for observations without enough data. "
+                yield "#' @param threshold This argument is deprecated, use `min_sdev` instead. The minimum standard deviation to use for observations without enough data. "
+                yield "#'                  Must be at least 1e-10."
```

Comment:
Hmm... Those were the only tests in R that I could find that use those parameters. Do we need a test for the valid parameters?